tutormfe/templates/mfe/build/mfe/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM docker.io/node:16.14-bullseye-slim AS base
+# https://hub.docker.com/_/node/tags
+FROM docker.io/node:16.20-bullseye-slim AS base
```

Comment:
MFEs included in palm have all been upgraded to Node 18 now. Can we bump up the node version here?

Reply:
Indeed! I managed to build the image locally. Now building on the Palm demo platform.